packages/font-glyphs/src/symbol/geometric/masked.ptl

```diff
@@ -449,3 +449,28 @@ glyph-block Symbol-Geometric-Masked : for-width-kinds WideWidth1
 			include [refer-glyph : MangleName 'whiteTriangleUp'] AS_BASE ALSO_METRICS
 			include : HSerif.lb (Geom.MidX - triangleSideRadius) Geom.Bot jut GeometryStroke
 			include : HSerif.rb (Geom.MidX + triangleSideRadius) Geom.Bot jut GeometryStroke
+
+		create-glyph [MangleName 'marriageSymbol'] [MangleUnicode 0x26AD] : glyph-proc
+			set-width Geom.Width
+			local offsetX : Geom.Size * Size.SlightlySmall.size / 2
+			include : union
+				with-transform [Translate offsetX 0] : refer-glyph : MangleName 'whiteSlightlySmallCircle'
```

Comment:
Need to make the ring thinner at Heavy.

Reply:
Since all the partnership symbols refer to one symbol, I'd prefer not to make a separate ring just for `⚭`. Instead I'm just spacing the gaps more evenly for now.

If this doesn't solve it, it's probably better to use `⚬` directly for `⚭`, at the expense of making it differently sized from `⚮⚯`.